style.css

```diff
@@ -21,11 +21,8 @@ body {
 }
 
 .header {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
   color: #6070ff;
-  padding: 50px 20px;
+  padding: 20px;
```

Comment:
- [ ] Please, consider that this padding gives a white space for your mobile menu. Try to remove it and you will see that the menu cover is without white spaces 👍 
![image](https://user-images.githubusercontent.com/49064106/188680432-55079ae7-94cd-4a06-a3f2-e199ffeb137f.png)

> your CSS
![image](https://user-images.githubusercontent.com/49064106/188680722-d119e57b-fd6a-48be-9c9d-f4aa0c677e88.png)